components/intake.py

```diff
@@ -0,0 +1,42 @@
+import ctre
+import wpilib
+import enum
+
+
+class Action(enum.Enum):
+    Intake = 1
+    Outtake = -1
+    Stop = 0
+
+
+class Intake:
+    """Bunny intake"""
+    intake_motor = ctre.CANTalon
+    intake_pdp_channel = int
+    pdp = wpilib.PowerDistributionPanel
+    # Max current in amps to draw before stopping the motor
+    max_current = 6
+    releaseBunny = False
+    holdingBunny = False
```

Comment:
Is `has_bunny` a better name?

Reply:
It should using an underscore, but I don't really have a preference between `holding_bunny` and `has_bunny`, whatever you want.

Reply:
I prefer has_bunny